extensions/amp-story/0.1/amp-story.css

```diff
@@ -472,10 +489,18 @@ amp-story-grid-layer * {
 .i-amphtml-story-bookend-article-set {
```

Comment:
Everything here down should be `!important` or moved to `amp-story-user-overridable.css`